extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -104,8 +104,8 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 		return Project.openProject(projectFilePath);
 	}
 
-	public promptAddItem(project: sqldbproj.ISqlProject, relativeFilePath: string, itemType?: string): Promise<void> {
-		return this.projectController.addItemPrompt(project, relativeFilePath, itemType);
+	public promptAddItem(project: sqldbproj.ISqlProject, relativeFilePath: string, options?: sqldbproj.AddItemOptions): Promise<void> {
```

Comment:
any reason why this is called promptAddItem instead of addItemPrompt, like the function it calls? I missed noticing this in the previous PR

Reply:
Nope, other than I thought promptAddItem (we're prompting to add an item) read better. Happy to change it if you would prefer to be consistent. (or maybe change `addItemPrompt` instead)

Reply:
I'd prefer for them to be the same. No preference on which one you change :)

Reply:
Alrighty, changed to match original so that it also keeps in line with the other *Prompt() functions (such as `addExistingItemPrompt`)